conf/puppet.conf

```diff
@@ -0,0 +1,9 @@
+[main]
+    # Where Puppet stores dynamic and growing data.
+    vardir=/opt/puppetlabs/agent/cache
+
+    # The Puppet log directory.
+    logdir=/var/log/puppetlabs/agent
+
+    # Where Puppet PID files are kept.
+    rundir=/var/run/puppetlabs
```

Comment:
Yeah, it would appear that on Windows acceptance runs from git, these configuration values are getting picked up and used by Windows.

What necessitated the addition of this file to the PR?